### ISSUE TITLE
Integration with Ionic Framework

### ISSUE THREAD

**Issue by Azim212** [Tastenkunst/brfv4_javascript_examples#67]
Is it possible to get this working in an Ionic Framework and to deploy onto mobile applications using Cordova? It is a fantastic library that I wish to use in my Ionic projects.

**Comment by Azim212** [comment]
I actually got it to run on my android device using Ionic but the app is unable to detect the camera. Is there a way to change where the video's source is coming from?

**Comment by MarcelKlammer** [comment]
The camera handling is the reason I never implemented Cordova. There are some native camera plugins out there, but didn't have a proper look in ages.

Also BRFv4 isn't supported anymore.

BRFv5 is the current version.

Cheers
Marcel

**Comment by Azim212** [comment]
I see. Thanks for the reply! 

Actually, I managed to get it to work running on android/ WebView. The problem was simply because of android permissions in opening the camera. There was a way to show a dialog box to the user to accept the camera permissions but I was more focused on seeing the demo run, so I just allowed the camera permissions in the settings on my phone. 

For anyone else curious, this page helped me out a lot : [https://stackoverflow.com/questions/48775154/notreadableerror-could-not-start-source](url)
